services/training/devops-training.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: training
+training: Zero to Continuous Delivery with Docker
+---
+
+# Zero to Continuous Delivery with Docker - 2 Days
+
+## About
+
+Are you a developer looking to foster a DevOps culture at your workplace? This course will take attendees right from the basics of containers and tools like Docker, all the way up to clustered infrastructures supporting continuous deployment, horizontal scalability and zero downtime. The course will be hands-on and attendees will be building their own solutions on [AWS](http://aws.amazon.com/).
+
+## Why attend?
+
+* Understand the principles underpinning containers and tools like Docker
+* Learn how to build and monitor your own continuous delivery pipeline
+* Set up and deploy to your own remotely hosted container cluster
+
+## Content
+
+* Introduction to Docker and the Docker CLI
+* Build, run and publish your own Docker images
+* Deploy containers to a remote server
+* Set up a continuous delivery pipeline
+    * Automate build and deployment as part of your continuous integration process using Jenkins
+    * Establish healthchecks for your services using Nagios
+    * Implement a service discovery solution using tools like Consul and Nginx
+* Set up a cluster of Docker daemons using Docker Swarm
+* Explore techniques and strategies for achieving zero downtime
```

Comment:
I feel that having the tools specified would help people to have a better idea about the course. Helps with searches on Google as well.